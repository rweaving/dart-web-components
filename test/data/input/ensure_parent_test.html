<!doctype html>
<!--
   Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
   for details. All rights reserved. Use of this source code is governed by a
   BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script type='application/javascript' src="testing.js"></script>
  <style> .blue { color: #0000FF; } .red { color: #FF0000; } </style>
</head>
<body>
  <!-- regression test for issue #174. -->
  expect list with 2 bullets, one blue (with blue color style), one red (with
  red color style):
  <ul>
    <template iterate="item in list">
      <li><span class={{item}}>{{item}}</span></li>
    </template>
  </ul>
  <script type="application/dart">
    import 'dart:html';
    import 'package:web_components/watcher.dart';
    import 'common.dart';
    var list = ["blue", "red"];
    main() {
      window.setTimeout(() => window.postMessage('done', '*'), 0);
    }
  </script>
</body>
</html>
