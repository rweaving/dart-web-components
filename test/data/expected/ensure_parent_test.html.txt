Blocked access to external URL http://dart.googlecode.com/svn/branches/bleeding_edge/dart/client/dart.js
Blocked access to external URL http://dart.googlecode.com/svn/branches/bleeding_edge/dart/client/dart.js
Content-Type: text/plain
layer at (0,0) size 800x600
  RenderView at (0,0) size 800x600
layer at (0,0) size 800x100
  RenderBlock {HTML} at (0,0) size 800x100
    RenderBody {BODY} at (8,8) size 784x76
      RenderBlock (anonymous) at (0,0) size 784x20
        RenderText {#text} at (0,0) size 522x19
          text run at (0,0) width 428: "expect list with 2 bullets, one blue (with blue color style), one red (with "
          text run at (428,0) width 94: "red color style):"
      RenderBlock {UL} at (0,36) size 784x40
        RenderListItem {LI} at (40,0) size 744x20
          RenderListMarker at (-18,0) size 7x19: bullet
          RenderInline {SPAN} at (0,0) size 25x19 [color=#0000FF]
            RenderText {#text} at (0,0) size 25x19
              text run at (0,0) width 25: "blue"
        RenderListItem {LI} at (40,20) size 744x20
          RenderListMarker at (-18,0) size 7x19: bullet
          RenderInline {SPAN} at (0,0) size 20x19 [color=#FF0000]
            RenderText {#text} at (0,0) size 20x19
              text run at (0,0) width 20: "red"
      RenderBlock (anonymous) at (0,92) size 784x0
        RenderText {#text} at (0,0) size 0x0
#EOF
